### PR TITLE
resubmit changes

### PR DESCRIPTION
**Pull request by j-lam**

### CODE REVIEW

File: fall-2017/jennifer/swagger/swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+basePath: /v2
+tags:
+  - name: task
+    description: Everything about tasks
+  - name: user
+    description: Everything about user
+schemes:
+  - http
+paths:
+  /user/task:
```
Comment (toddysm) [inline fall-2017/jennifer/swagger/swagger.yaml]: This is a wrong path - we don't have /user/task path

File: fall-2017/jennifer/swagger/swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+          schema:
+            $ref: '#/definitions/Task'
+      responses:
+        '400':
+          description: Invalid ID supplied
+        '404':
+          description: Pet not found
+        '405':
+          description: Validation exception
+  '/user/task/{taskId}':
```
Comment (toddysm) [inline fall-2017/jennifer/swagger/swagger.yaml]: This is also wrong - I think you mean /users/{userid}/tasks/{taskid}

File: fall-2017/jennifer/swagger/swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+          description: User object that needs to be added to the
+          required: false
+          schema:
+            $ref: '#/definitions/User'
+      responses:
+        '200':
+          description: OK
+        '405':
+          description: Invalid input
+  /user/login:
```
Comment (toddysm) [inline fall-2017/jennifer/swagger/swagger.yaml]: We don't have path /user/login

File: fall-2017/jennifer/swagger/swagger.yaml [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+              type: integer
+              format: int32
+              description: calls per hour allowed by the user
+            X-Expires-After:
+              type: string
+              format: date-time
+              description: date in UTC when token expires
+        '400':
+          description: Invalid username/password supplied
+  /user/logout:
```
Comment (toddysm) [inline fall-2017/jennifer/swagger/swagger.yaml]: ...As well as /user/logout

**Comment by j-lam** [comment]
resubmitting with changes

**Pull request closed without merging**